[chore][exporter/elasticsearch] docs: Improve version conflict known issue (#39420)
open-telemetry/opentelemetry-collector-contrib: exporter/elasticsearchexporter/README.md

The exporter can only group metrics with the same dimensions into the same document if they arrive in the same batch.
To ensure metrics are not dropped even if they arrive in different batches in the exporter, the exporter adds a fingerprint of the metric names to the document in the `otel` mapping mode.

> Note that you'll need to be on a minimum version of Elasticsearch in order for this to take effect 8.17.6, 8.19.0, 9.0.0.

Reviewer: ```suggestion
Note that this functionality requires a minimum Elasticsearch version of 8.17.6, 8.19.0, or 9.0.0.
```
This line sounds confusing to me. Maybe the above edit works?

Author: thanks!

}
```

> After installing this component template, if you have previously ingested data, you'll have to wait until the time series data stream's old index's `end_time`. It may take up to 30 minute by default, see [time series data stream look ahead time](https://www.elastic.co/guide/en/elasticsearch/reference/current/tsds-index-settings.html).

Reviewer: ```suggestion
After installing this component template, if you've previously ingested data, you'll need to wait until the old index of the time series data stream reaches its `end_time`. This can take up to 30 minutes by default. See [time series data stream look ahead time](https://www.elastic.co/guide/en/elasticsearch/reference/current/tsds-index-settings.html) for more information.
```

Does this sound better?
